stablehlo/transforms/StablehloLegalizeToVhlo.cpp

```diff
@@ -268,8 +270,10 @@ class StablehloToVhloOpConverter : public OpConversionPattern<StablehloOpTy> {
     SmallVector<NamedAttribute> vhloAttrs;
     for (NamedAttribute stablehloAttr : stablehloOp->getAttrs()) {
       Attribute vhloAttr;
-      if constexpr (std::is_same<StablehloOpTy,
-                                 stablehlo::CustomCallOp>::value) {
+      if (stablehloAttr.getName() == "use_global_device_ids") {
```

Comment:
Do we want to check if it is a UnitAttr?

Reply:
Done.